geotrek/maintenance/tests/test_filters.py

```diff
@@ -97,6 +97,30 @@ def test_filter_zoning_area_type(self):
         self.assertEqual(len(filter.qs), 0)
 
 
+class InterventionDateFilterTest(TestCase):
+
+    def test_filter_year_without_end_date(self):
+        InterventionFactory(name="intervention1", begin_date="2020-07-30")
+        InterventionFactory(name="intervention1", begin_date="2021-07-30")
+        InterventionFactory(name="intervention1", begin_date="2022-07-30")
+        intervention_filter = InterventionFilterSet({'year': [2021]})
+        self.assertEqual(len(intervention_filter.qs), 1)
```

Comment:
```suggestion
        self.assertCountEqual(intervention_filter.qs, 1) # or self.assertEqual(intervention_filter.qs.count(), 1)
```
on all tests will improve test perfs by executing COUNT instead of complete SQL query